### DISCUSSION TITLE
Is https mandatory for remote files?

### DISCUSSION THREAD

**Discussion by nedd0** [bluefireteam/audioplayers#1066] [Q&A] [answered]
Hello! 

I'm doing an app and I'm still testing in the Android emulator in my computer. I included the audio player widget that comes as an example in this repo. It works fine if I use the same file used in the repo with an https url. When I use a file served by a django development server in my computer (that uses http) it doesn't work. If access the url directly it does download the mp3. I did change the url in the emulator to use 10.0.2.2 instead of localhost. I know that works because I can access an API in the same django server with no issues. 

I've been struggling for a couple of days with this, I guess the problem lies most likely on the Django side, but... is there any limitation in audioplayers for this scenario? IE: only https, can't use the same computer to run both, etc? 

Thanks!

**Comment by erickzanardo** [comment, marked as answer]
The limitation is actually on the operating system side, both android and iOS don't enable non https requests by default.

That can be changed though, we have a small section on out README.md (or FAQ.md) that showns how to change that.

**Comment by nedd0** [comment]
Thanks a lot!
